README.md

```diff
@@ -52,7 +52,7 @@ platform :ios, '9.0'
 use_frameworks!
 
 target 'MyApp' do
-  pod 'Cloudinary', '~> 3.0'
+  pod 'Cloudinary', '~> 3.1.0'
```

Comment:
We don't want to update this number on every new(minor/patch) version released, let's reconsider this.